genesyscloud/resource_genesyscloud_telephony_providers_edges_site.go

```diff
@@ -282,6 +328,11 @@ func createSite(ctx context.Context, d *schema.ResourceData, meta interface{}) d
 		return diag.Errorf("Location with id %v does not have an emergency number", locationId)
 	}
 
+	err = validateMediaRegions(mediaRegions, sdkConfig)
```

Comment:
This is the new validator function I added to the code.  I could not make a validator function in the schema yet as the Terraform framework does not support validation on lists and sets.